js/libs/form.js

```diff
@@ -344,11 +404,19 @@ Fliplet().then(function() {
               value = value.slice(0);
             }
 
-            if (typeof field.defaultValueSource !== 'undefined' && field.defaultValueSource !== 'default' && !entryLoaded) {
+            if (typeof field.defaultValueSource !== 'undefined' && field.defaultValueSource === 'profile' && !entryLoaded) {
```

Comment:
Added switch case. We have added entryLoaded for profile case because it will set value for all fields later in getFields() method, so it should not run again and again for profile case.